test/e2e-test-application/src/luigi-config/extended/globalSearch.js

```diff
@@ -1,20 +1,55 @@
 class GlobalSearch {
+  constructor() {
+    this.searchResult = [];
+  }
+
   searchProvider = {
     onInput: () => {
-      console.log('searchValue ', Luigi.globalSearch().getSearchString());
-      // let searchResultItem = {
-      //   pathObject: {
-      //     path,
-      //     params: {} // can be used by linkmanager.navigate(path).withParams(params)
-      //   },
-      //   label,
-      //   description,
-      //   onActivate() {}
-      // };
-      //Luigi.globalSearch().showSearchResult([searchResultItem]);
+      if (Luigi.globalSearch().getSearchString() === '') {
+        this.searchResult = [];
+        Luigi.globalSearch().closeSearchResult();
+      }
```

Comment:
I got a little too excited with the search 😄 . The autocomplete here looked super cool tho. Just a thought, might be too much in detail for this task anyway, skip most probably:

```
      this.searchResult = [];
      if (Luigi.globalSearch().getSearchString() === '') {
        Luigi.globalSearch().closeSearchResult();
        return; // skip extra work
      }
      let nodes = Luigi.getConfigValue('navigation.nodes');
      nodes
        .filter(
          element =>
            element.label &&
            element.label.toLowerCase().includes(Luigi.globalSearch().getSearchString().toLowerCase())
        )
        .forEach(result => {
          this.searchResult.push({
            pathObject: {
              path: result.pathSegment ? result.pathSegment : '',
              params: {} // can be used by linkmanager.navigate(path).withParams(params)
            },
            label: result.label,
            description: result.viewUrl ? result.viewUrl : '',
            onActivate() {}
          });
        });
      if (this.searchResult.length > 0) {
        Luigi.globalSearch().showSearchResult(this.searchResult);
      } else {
        Luigi.globalSearch().showSearchResult([
          {
            pathObject: {
              path: '',
              params: {} // can be used by linkmanager.navigate(path).withParams(params)
            },
            label: 'Nothing found',
            description: '',
            onActivate() {}
          }
        ]);
      }
```